damusTests/TimeAgoTests.swift

```diff
@@ -18,11 +18,11 @@ final class TimeAgoTests: XCTestCase {
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-2), calendar), "now")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3), calendar), "3s")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-59), calendar), "59s")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-60), calendar), "1min")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3599), calendar), "59min")
```

Comment:
Moved away from using seconds for time at the hour and beyond mark to more easily reason about and avoid needing to do manual calculations when writing these assertions.